jbehave-support-core/src/main/java/org/jbehavesupport/core/verification/VerificationSteps.java

```diff
@@ -0,0 +1,48 @@
+package org.jbehavesupport.core.verification;
+
+import java.util.List;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import org.assertj.core.api.SoftAssertions;
+import org.codehaus.plexus.util.StringUtils;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.model.ExamplesTable;
+import org.jbehavesupport.core.internal.parameterconverters.ExamplesEvaluationTableConverter;
+import org.jbehavesupport.core.internal.verification.VerifierNames;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.DATA;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.EXPECTED_VALUE;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.VERIFIER;
+import static org.jbehavesupport.core.internal.ExamplesTableUtil.convertTable;
+
+@Component
+@RequiredArgsConstructor
+public class VerificationSteps {
+
+    @Value("${verifier.max.assert.count:10}")
+    private int maxSoftAssertCount;
+
+    private final VerifierResolver verifierResolver;
+    private final ExamplesEvaluationTableConverter tableConverter;
+
+    @Then("following data are compared: $stringTable")
+    public void compareRows(String stringTable) {
```

Comment:
Why do you use string, isn't `ExamplesTable` better so you don't need to do conversion explicitly, but leave if on jbehave?